bodies/views.py

```diff
@@ -88,6 +92,22 @@ def follow(self, request, pk):
 
         return Response(status=204)
 
+    def get_events(self, request, pk):
+        """Get all events from pk uuid or strid.
+        {?archived} is optional arguement to fetch all events"""
+        body = self.get_body(pk)
+
+        # Get query param
+        archived = 'archived' in request.GET
+
+        if archived:
+            queryset = Event.objects.filter(bodies=body).order_by('-start_time')
+        else:
+            last_year = date.today() + relativedelta(years=-1)
```

Comment:
I'll suggest use `from django.utils import timezone` and `from datetime import timedelta` instead of `date` and `relativedelta` for two reasons:
1) Django understands these better and takes better care of of database format conversion etc.
2) Consistency with the rest of the code base - makes it easier to fix failing library updates in the future